pkg/edgeview/build.yml

```diff
@@ -1,6 +1,5 @@
 image: eve-edgeview
 org: lfedge
-network: yes
```

Comment:
Curious, how it works here. We do not use vendoring for edgeview now as Milan [mentioned](https://github.com/lf-edge/eve/pull/2884#issue-1422139950).

Reply:
So we might get an error here as well. This is not ready for review yet, as I have not completely disabled network, just showing how. I will do that in the next few days.

Reply:
Got it, thanks!